static/app/components/events/groupingInfo/index.tsx

```diff
@@ -31,7 +32,15 @@ type State = AsyncComponent['state'] & {
 
 class GroupingInfo extends AsyncComponent<Props, State> {
   getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
-    const {organization, event, projectSlug} = this.props;
+    const {organization, event, projectSlug, group} = this.props;
+
+    if (
+      event.occurrence &&
+      group?.issueCategory === IssueCategory.PERFORMANCE &&
+      event.type === 'transaction'
```

Comment:
checking for `event.type` here explicitly to handle performance issue case since profiling issues are also of performance category